.github/workflows/build.yml

```diff
@@ -368,8 +368,8 @@ jobs:
           bundle exec fastlane run validate_play_store_json_key
 
       - name: Publish Play Store bundle
-        if: ${{ matrix.variant == 'prod' && matrix.artifact == 'aab' && inputs.publish-to-play-store }}
-        run: bundle exec fastlane publishForInternalTesting
```

Comment:
Do we need some other task to publish it for internal testing still?

Reply:
No. At the moment we're only publishing to the Public Beta. Internal testing will be done using the internal Firebase distributions.

Reply:
OK